README.md

```diff
@@ -63,7 +66,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+rules of scope for Javascript are complicated...it is the context of the code which can be global or local. It is determined by the accessibility of variables and functions and determining when and where they ARE functional within the context of the code...ish.
 ```
```

Comment:
Not that complicated! If something's declared inside a function, it's accessible anywhere inside that function, but not outside that function. Pretty much it.